sim-cli/src/main.rs

```diff
@@ -46,6 +48,9 @@ struct Cli {
     /// Do not create an output file containing the simulations results
     #[clap(long, default_value_t = false)]
     no_results: bool,
+    /// The interval at which to log payment summaries, expressed in seconds.
+    #[clap(long, default_value_t = 60)]
```

Comment:
nit: I think we should define this as a constant, given all the other defaults are.

We can also take the chance to remove the line break between them :P